pkg/operator/k8sutil/labels.go

```diff
@@ -45,3 +46,15 @@ func ParseStringToLabels(in string) map[string]string {
 
 	return labels
 }
+
+// AddRecommendedLabels adds the labels to the resources created by rook
+// The labels added are name, unique_instance,etc
+func AddRecommendedLabels(labels map[string]string, appName, resourceName, resourceKind, resourceInstance string) {
```

Comment:
How about we call this the parentName? Then it's clear we want the name of the CR that owns this resource.
```suggestion
func AddRecommendedLabels(labels map[string]string, appName, parentName, resourceKind, resourceInstance string) {
```